di_website/datasection/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Please delete this file. Whenever you start a new app in Wagtail, you generally don't need `admin.py` `tests.py` or `views.py`.

Reply:
Just deleted all three.